tests/primer3_suite.xml

```diff
@@ -1,4 +1,4 @@
-<suite name="Primer 3 tests" test-timeout="30">
+<suite name="Primer 3 tests" test-timeout="300">
```

Comment:
Do we really have individual tests that runs for 5 minutes?

Reply:
Yes, there are a bunch of time-consuming tests, like `lib_amb_codes (08-11)` and other tests (do not remember them all, just run the whole suite to figure out)